db/migrate/20230607103526_create_transfer_tasks_data.rb

```diff
@@ -0,0 +1,7 @@
+class CreateTransferTasksData < ActiveRecord::Migration[7.0]
+  def change
+    create_table :transfer_task_lists, id: :uuid do |t|
```

Comment:
Can the table be `transfer_tasks_data` then we don't need the table_name? We need to update the conversion one as well.